manifests/proxydhcp.pp

```diff
@@ -72,12 +72,15 @@
 
     ensure_packages(['grep', 'acl'])
 
-    [$dhcp::dhcp_dir, dirname($foreman_proxy::dhcp_leases)].each |$path| {
+    [$dhcp::dhcp_dir, "$dhcp::dhcp_dir/dhcpd.conf", dirname($foreman_proxy::dhcp_leases)].each |$path| {
```

Comment:
You should quote
```suggestion
    [$dhcp::dhcp_dir, "${dhcp::dhcp_dir}/dhcpd.conf", dirname($foreman_proxy::dhcp_leases)].each |$path| {
```

However, I wonder if the require on the DHCP class is sufficient.

Reply:
Yes, thank you